src/components/Context.ts

```diff
@@ -15,17 +15,20 @@ export interface ReactReduxContextValue<
   noopCheck: CheckFrequency
 }
 
-const ContextKey = Symbol.for(`react-redux-context-${ReactVersion}`)
-const gT = globalThis as { [ContextKey]?: Context<ReactReduxContextValue> }
+const ContextKey = Symbol.for(`react-redux-context`)
+const gT = globalThis as {
```

Comment:
Per https://github.com/reduxjs/react-redux/pull/2039#pullrequestreview-1524058384 , can we do some more checks to see if `globalThis` even exists?

Reply:
Given that Android 8.1 mentioned in that comment had it's official support from Google end on October 4, 2021, I would be very happy if we could avoid that - or at least only add it temporarily and remove it in a 9.0 release.